www/FabricPlugin.Crashlytics.es6

```diff
@@ -19,6 +19,11 @@ class FabricCrashlytics {
         ]);
     }
 
+    recordError() {
+        window.fabric.core.execPlugin('recordError', [
```

Comment:
The document says `recordError` accepts a message parameter, but it is not passed here.